client/src/holdings/AddTradeDialog.tsx

```diff
@@ -30,6 +33,26 @@ export function AddTradeDialog({
   showNotification,
 }: AddTradeDialogProps) {
   const [open, setOpen] = useState(false);
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const handleDateChange = (date: any) => {
+    setSelectedDate(date);
+  };
```

Comment:
Do you need these? It should all be possible within the react-hook-form methods.